source/braille.py

```diff
@@ -1677,7 +1690,16 @@ def handleGainFocus(self, obj):
 
 	def _doNewObject(self, regions):
 		self.mainBuffer.clear()
+		focusToHardLeftSet = False

```

Comment:
> Would the "functional" purpose of this be clearer if it were named, for example,  foundHardLeft ?

Honestly I don't think so, as we aren't only checking for focusToHardLeft on a region, but also setting it when the focus ancestry is equal.